### ISSUE TITLE
Error when recompiling css assets on Windows

### ISSUE THREAD

**Issue by byungjikroh** [gobuffalo/buffalo#893]
I got an error when liveload recompiles css assets after editing and can't load css file.

I guess rimraf module got an EBUSY error when it tries to remove previous asset files.

rimraf (https://github.com/isaacs/rimraf)
> Windows:  EBUSY and ENOTEMPTY - rimraf will back off a maximum of opts.maxBusyTries times before giving up, adding 100ms of wait between each attempt. The default maxBusyTries is 3.

It's an error only on Windows but I can't figure out how to fix it. 

```
C:\Users\Byungjik\go\src\example\node_modules\rimraf\rimraf.js:313
        throw er
        ^

Error: EBUSY: resource busy or locked, unlink 'C:\Users\Byungjik\go\src\example\public\assets\manifest.json'
    at Error (native)
    at Object.fs.unlinkSync (fs.js:1089:18)
    at rimrafSync (C:\Users\Byungjik\go\src\example\node_modules\rimraf\rimraf.js:306:17)
    at C:\Users\Byungjik\go\src\example\node_modules\rimraf\rimraf.js:342:5
    at Array.forEach (native)
    at rmkidsSync (C:\Users\Byungjik\go\src\example\node_modules\rimraf\rimraf.js:341:26)
    at rmdirSync (C:\Users\Byungjik\go\src\example\node_modules\rimraf\rimraf.js:334:7)
    at Function.rimrafSync [as sync] (C:\Users\Byungjik\go\src\example\node_modules\rimraf\rimraf.js:304:9)
    at C:\Users\Byungjik\go\src\example\node_modules\clean-webpack-plugin\index.js:164:16
    at Array.forEach (native)
    at CleanWebpackPlugin.clean (C:\Users\Byungjik\go\src\example\node_modules\clean-webpack-plugin\index.js:90:15)
    at Compiler.<anonymous> (C:\Users\Byungjik\go\src\example\node_modules\clean-webpack-plugin\index.js:188:15)
    at Compiler.applyPlugins (C:\Users\Byungjik\go\src\example\node_modules\tapable\lib\Tapable.js:61:14)
    at C:\Users\Byungjik\go\src\example\node_modules\webpack\lib\Compiler.js:479:8
    at Compiler.applyPluginsAsyncSeries (C:\Users\Byungjik\go\src\example\node_modules\tapable\lib\Tapable.js:195:46)
    at Compiler.compile (C:\Users\Byungjik\go\src\example\node_modules\webpack\lib\Compiler.js:476:7)
    at C:\Users\Byungjik\go\src\example\node_modules\webpack\lib\Compiler.js:46:17
    at next (C:\Users\Byungjik\go\src\example\node_modules\tapable\lib\Tapable.js:202:11)
    at Compiler.compiler.plugin (C:\Users\Byungjik\go\src\example\node_modules\webpack\lib\CachePlugin.js:32:5)
    at Compiler.applyPluginsAsyncSeries (C:\Users\Byungjik\go\src\example\node_modules\tapable\lib\Tapable.js:206:13)
    at Watching._go (C:\Users\Byungjik\go\src\example\node_modules\webpack\lib\Compiler.js:44:16)
    at Watching.invalidate (C:\Users\Byungjik\go\src\example\node_modules\webpack\lib\Compiler.js:125:8)
    at Watching.<anonymous> (C:\Users\Byungjik\go\src\example\node_modules\webpack\lib\Compiler.js:109:8)
    at EventEmitter.watcher.once (C:\Users\Byungjik\go\src\example\node_modules\webpack\lib\node\NodeWatchFileSystem.js:45:4)
    at EventEmitter.g (events.js:291:16)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:191:7)
    at EventEmitter._onTimeout (C:\Users\Byungjik\go\src\example\node_modules\watchpack\lib\watchpack.js:142:7)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
```

**Comment by markbates** [comment]
I'm not sure this is an issue with Buffalo. All Buffalo does is run web pack, it doesn't compile or watch any javascript on its own. Have you tried seeking help with rimraf?

**Comment by byungjikroh** [comment]
Yes, I'm digging rimraf codes now.

rimraf.js:313 line is 'if (er.code !== "EISDIR")' and EISDIR error occurs when it tries to open a file, but the path given is a directory.

I opened issue for others. I will leave a comment when I find a solution. 

Thanks for fast reply. :)

**Comment by markbates** [comment]
@byungjikroh I think I found the problem. It was actually in the https://github.com/gobuffalo/packr tool. I was able to reproduce and fix it, hopefully, I'm not a Windows dev. :) 

The fix is in v1.10.3 of packr. Update to that and you should be all set!

https://github.com/gobuffalo/packr/releases/tag/v1.10.3

**Comment by byungjikroh** [comment]
After updating packr tool, it works without an error. 
Thanks for updating. :)
